waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/MetaStoreUriNormaliserTest.java

```diff
@@ -0,0 +1,32 @@
+package com.hotels.bdp.waggledance.client;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class MetaStoreUriNormaliserTest {
+
+  @Test
+  public void normaliseValidUris() {
+    MetaStoreUriNormaliser normaliser = new MetaStoreUriNormaliser();
+    String firstUri = "thrift://localhost:1";
```

Comment:
You should try pass in a uri string that actually gets changed in some way by the normalise method. Let me know if you can't figure out one.